package.json

```diff
@@ -12,7 +12,7 @@
   },
   "homepage": "https://remotestorage.io",
   "scripts": {
-    "test": "./scripts/test",
+    "test": "bash scripts/test-all.sh",
```

Comment:
Why the `bash` prefix? What about people using a different shell?

Reply:
That's the reason, so that it doesn't break when they use a different shell. The script is written for bash. Also, so that it's entirely clear what is being run without looking into the script first.

Reply:
Ah, makes sense.

Reply:
The script is executable and specifies `/bin/bash` in the shebang so you don't need to specify a shell, but it doesn't hurt. In this case we could have used `/bin/sh` because our script doesn't contain bashisms, I forgot to check when I extracted it from the Makefile

Reply:
I think it's a good idea to just go with just one as baseline, and bash is common enough to be that imo.